src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -36,6 +40,7 @@ interface ReviewState {
   errorModalCtaAction: () => null
 }
 
+@track()
```

Comment:
Do we need this empty `@track()`? Not sure if it has some inherent meaning.

Reply:
IIRC @xtina-starr mentioned that that should be in there when using tracking, but I'm not sure what purpose it has. @alloy do you have thoughts here?

Reply:
Yep, this is needed to pull the tracking functions out of the React context. It’s essentially a  context HOC wrapped in a class decorator.